.github/workflows/release-artifacts.yml

```diff
@@ -24,20 +24,12 @@ jobs:
       group: build-runtime-${{ matrix.runtime }}-${{ github.ref }}
       cancel-in-progress: true
     container:
-      image: ubuntu:latest
+      image: niteo/nixpkgs-nixos-22.05:316b762afdb9e142a803f29c49a88b4a47db80ee
     strategy:
       matrix:
         runtime: [dali, picasso, composable]
     steps:
-      - run: |
-          apt-get update
-          apt-get install -y curl git
-          curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | bash
-          apt-get install git-lfs
       - uses: actions/checkout@v3
-        with:
-          lfs: true
-          fetch-depth: 0
```

Comment:
This line may need to be kept as per original

Reply:
I'm 99% sure not, but I can re-add the fetch-depth:0

Reply:
Yeah was checking the docs, just trying to be safe